src/Symfony/Component/Scheduler/Schedule.php

```diff
@@ -83,4 +113,9 @@ public function getSchedule(): static
     {
         return $this;
     }
+
+    public function restart(bool $restart = null): bool
```

Comment:
For more clarity, I would split this method: `setRestart(bool)` and `shouldRestart(): bool`